### PR TITLE
Aio structural directives small amends

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Issue

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information
 sorry just a couple of small things that slipped through the cracks in #44895 :sweat:

### CODE REVIEW

File: aio/content/guide/structural-directives.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This guide is about structural directives and provides conceptual information on
 
 Structural directives are directives which change the DOM layout by adding and removing DOM element.
 
-Angular provides a set of built-in structural directives (such as `NgIf`, `NgFor`, `NgSwitch` and others) which are commonly used in all Angular projects. For more information see [Built-in directives](guide/built-in-directives).
+Angular provides a set of built-in structural directives (such as `NgIf`, `NgForOf`, `NgSwitch` and others) which are commonly used in all Angular projects. For more information see [Built-in directives](guide/built-in-directives).
```
Comment (dario-piotrowicz) [inline aio/content/guide/structural-directives.md:7]: I noticed this as it breaks the auto-linking:
Screenshot at 2022-06-02 21-17-30

( compared to:
Screenshot at 2022-06-02 21-20-33

)

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

reviewed-for: public-api, fw-core

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: public-api

**Comment by alxhub** [comment]
This PR was merged into the repository by commit ee98f4eaf844c84bbdbc93bad62fb574fa4c8589.

**Pull request closed without merging**